docs/release-notes/2.33.0.md

```diff
@@ -0,0 +1,147 @@
+# Percona Monitoring and Management 2.33.0
+
+| **Release date:** | Dec 13, 2022                                                                                    |
+| ----------------- | ----------------------------------------------------------------------------------------------- |
+| **Installation:** | [Installing Percona Monitoring and Management](https://www.percona.com/software/pmm/quickstart) |
+
+Percona Monitoring and Management (PMM) is an open source database monitoring, management, and observability solution for MySQL, PostgreSQL, and MongoDB.
+
+!!! caution alert alert-warning "Important"
+    We recommend using the latest version of PMM. This ensures that you have access to the latest PMM features and that your environment runs on the latest version of the underlying components, such as VictoriaMetrics, with all the bug fixes in place.
+
+## Release Highlights
+
+
+### Data loss prevention through VictoriaMetrics integration
+
+If PMM Server goes down, metrics will be buffered on the monitored instance until PMM Server comes back online, which is when metrics will get pushed to PMM Server. This only applies to PMM Clients which run in the `push mode`. The same holds true when a brief network outage occurs on the client side.
+
+The first step towards HA in PMM is the implementation of this improvement. Furthermore, we are happy to provide our users with a roadmap that includes the implementation of High Availability (HA) in PMM, a new solution they have been anticipating. In the event of a failover, your PMM setup will experience near-zero downtime, and monitoring will continue without any data loss.
+
+To ensure a smooth rollout, HA in PMM will be introduced in phases, in line with your requirements and feedback, with PMM 2.33.0 being the starting point. 
+
+!!! caution alert alert-warning "Important"
+    HA in PMM will be rolled out in three stages. Stage one of HA in PMM is the implementation to prevent data loss with VictoriaMetrics integration for short outages. HA stages two and three will be rolled out next year.
```

Comment:
```suggestion
    PMM HA solution will be rolled out in three stages. Stage one of PMM HA is the implementation to prevent data loss with VictoriaMetrics integration for short outages. HA stages two and three will be rolled out next year.
```